code/obj/artifacts/artifacts.dm

```diff
@@ -312,3 +312,9 @@
 		/datum/artifact/watercan,
 		/datum/artifact/pitcher
 		)
+
+/obj/artifact_type_spawner/war_engine
+	types = list(/datum/artifact/bomb/explosive,
+		/datum/artifact/bomb/explosive/devastating,
+		/datum/artifact/power_gen
+		)
```

Comment:
Seeing  as it is very easy to see if a prefab exists at the start of a round via the mining map, I could feasibly see people doing 

1. roundstart
2. check map
3. go to prefab if it exists
4. 33% chance of devastating artbomb within 3 minutes or so

Maybe consider making this a weighted list with reduced likelihood of a devastating artbomb, to make that a less likely target for powergaming?